config/test.exs

```diff
@@ -18,9 +18,6 @@ config :logger, level: :warn
 
 config :erlef, Erlef.Data.Repo,
   database: "erlef_website_test",
-  username: "postgres",
```

Comment:
Branch needs a rebase, but we do need this set for tests in ci.

Reply:
Should be up-to-date